walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java

```diff
@@ -268,6 +270,129 @@ public void notEnoughRoom() {
     Collection<TimeRange> actual = query.query(events, request);
     Collection<TimeRange> expected = Arrays.asList();
 
+    Assert.assertEquals(expected, actual);
+  }
+
+   @Test
+  public void optionalAttendeeBusyAllDay() {
+    // Have one optional person, but make it so that they can't make the meeting at any point in the day.
+
+    Collection<Event> events = Arrays.asList(
+        new Event("Event 1", TimeRange.fromStartDuration(TIME_0800AM, DURATION_30_MINUTES),
+            Arrays.asList(PERSON_A)),
+        new Event("Event 2", TimeRange.fromStartDuration(TIME_0900AM, DURATION_30_MINUTES),
+            Arrays.asList(PERSON_B)),
+        new Event("Event 3", TimeRange.WHOLE_DAY, Arrays.asList(PERSON_C)));
+
+    MeetingRequest request =
+        new MeetingRequest(Arrays.asList(PERSON_A, PERSON_B), DURATION_30_MINUTES);
+
+    request.addOptionalAttendee(PERSON_C);
+
+    Collection<TimeRange> actual = query.query(events, request);
+    Collection<TimeRange> expected =
```

Comment:
nit: we usually prefer to have tests set up in the following way: 
1) set up - set up the required variables
2) act - actually call the function being tested
3) assert - assert that your expected value is the same as actual

Because of this, I would move the definition of the `expected` collection  to right below line 290

Reply:
Thank you for clearing this up.